manifests/init.pp

```diff
@@ -228,8 +228,9 @@
   $krb5_conf_owner          = 'root',
   $krb5_conf_group          = 'root',
   $krb5_conf_mode           = '0444',
-  $alter_etc_services       = false
-) {
+  $alter_etc_services       = false,
+  $includedir               = $mit_krb5::params::includedir,
+) inherits mit_krb5::params {
```

Comment:
I'm afraid `inherits` is deprecated, and so is the `params.pp` paradigm.

Reply:
Obviously I'm broadcasting my lack of skill with GIT.  It was not my intention to include the inherits/params.pp design in my pull request last month.  I made the hackish inherits/params.pp code yesterday and didn't realize it'd get included in my pull request from last month.   

I tried copying the format/syntax in the current ../templates/*erb files but I'm no Ruby programmer and the logic behind the code escapes me.